### DISCUSSION TITLE
Welcome to react-native-iap Discussions!

### DISCUSSION THREAD

**Discussion by hyochan** [hyochan/react-native-iap#1315] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyochan** [comment]
Issue board is bad to solve problems because of duplication. I hope this discussion tab to be much valuable.

**Comment by basemanabulsi** [comment]
Hello Guys is there a way to get the localized descriptions? 

I tried to use `getSubscriptions` method with an IOS tester sandbox account in France region and the subscriptions have a french description localization. 
Is there any way to extract them or I should raise it as a bug?

**Reply by hyochan** [reply]
@basemanabulsi Could you kindly open up a new thread if you want to start the thread?

Also, what you've described here seems to work correctly if I am understanding right. What are you expecting?

**Reply by basemanabulsi** [reply]
@hyochan I tried this behavior but I am not getting the french description. I am getting the default english description for all region while I have a french description for example.

**Reply by hyochan** [reply]
Did you try to write `French` description in StoreKit? You may follow [Q & A section in readme](https://github.com/dooboolab/react-native-iap#getproducts-returns-empty-array).
